service/raft/raft_group0.hh

```diff
@@ -153,6 +153,10 @@ public:
     // We'll look for the other node's Raft ID in the group 0 configuration.
     future<> remove_from_group0(gms::inet_address host);
 
+    // Loads server address for group 0 from disk if present,
+    // otherwise randomly generates a new one and persists it.
+    // Execute on shard 0 only.
+    future<raft::server_id> load_or_create_my_id();
```

Comment:
The `load_or_create_my_addr()` member function is still declared (below `load_my_id()`) but now doesn't have an implementation.

Also, the comment should probably be updated: s/server address/server ID/

Reply:
Removed.